vendor/bat-native-ads/src/bat/ads/internal/covariates/log_entries/notification_ad_event_unittest.cc

```diff
@@ -0,0 +1,89 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/covariates/log_entries/notification_ad_event.h"
+
+#include <sstream>
+
+#include "bat/ads/confirmation_type.h"
+#include "bat/ads/internal/base/unittest/unittest_base.h"
+
+// npm run test -- brave_unit_tests --filter=BatAdsFederatedLogEntries*
+
+namespace {
+
+std::string GetValueForEventType(
+    const ads::mojom::NotificationAdEventType event_type) {
+  ads::NotificationAdEvent notification_ad_event;
+  notification_ad_event.SetEventType(event_type);
+  return notification_ad_event.GetValue();
+}
+
+std::string GetEventTypeAsString(
+    const ads::mojom::NotificationAdEventType event_type) {
```

Comment:
can you please add `namespace ads` to file and remove `ads::`, see other places too. Any files inside bat-native-ads should have `namespace ads`